skills/cut-the-crap/README.md

```
@@ -0,0 +1,46 @@
+```bash
+git clone https://github.com/breezy-bays-labs/crap4rs
+cp -r crap4rs/skills/cut-the-crap ~/.claude/skills/
+```
+
+Or, if you already have the repo checked out, run `cp -r skills/cut-the-crap ~/.claude/skills/` from the repo root.
+
+The skill is then invocable as `/cut-the-crap` from any Claude Code session — no restart required (skills are auto-discovered from `~/.claude/skills/`).
+
+## Usage
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)